connect_sample/src/main/java/com/zendesk/sample/connect/SampleMessagingService.java

```diff
@@ -0,0 +1,49 @@
+package com.zendesk.sample.connect;
+
+import android.app.Notification;
+import android.support.v4.app.NotificationCompat;
+
+import com.google.firebase.messaging.RemoteMessage;
+import com.zendesk.logger.Logger;
+
+import io.outbound.sdk.OutboundMessagingService;
+import io.outbound.sdk.PushNotification;
+
+/**
+ * You can optionally extend {@link OutboundMessagingService} to handle building
```

Comment:
Can you expand on the Javadoc here to explain what will happen if you don't extend `OutboundMessagingService`? Maybe explicitly call out that you need to register it in the AndroidManifest too.